problems/countNumbers.js

```diff
@@ -9,8 +9,16 @@
  * returns { 1:3, 2:2, 3:1, 4:1 }
  */
 
-function countNumbers() {
- 
+function countNumbers(arr) {
+    let count = {}
+    for(let i = 0; i < arr.length; i++){
```

Comment:
Looks good. Minor improvements. set count to const variable since we're not reassinging in. Use forEach.